salt/_beacons/metalk8s_kubeconfig_info.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from salt.ext.six.moves import map as _map
+
+__virtualname__ = 'metalk8s_kubeconfig_info'
+
+DEFAULT_NOTIFY_DAYS = 15
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def validate(config):
+    error_msg = "Configuration for {0} beacon is invalid".format(
+        __virtualname__
+    )
+
+    if not isinstance(config, list):
+        return False, "{0}: it must be a list".format(error_msg)
+
+    _config = {}
+    list(_map(_config.update, config))
```

Comment:
I guess this was copied from original Salt code :vomiting_face:. A cleaner, readable approach, would use a `for` loop:

```suggestion
    _config = {}
    for config_item in config:
        _config.update(config_item)
```

Reply:
Yes, it's what is done in each and every beacons, I wanted to do it the same way, but I can change it if you think it's too dirty. :)

Reply:
Changed with a for loop :)